### ISSUE TITLE
Put Release Notes on Website

### ISSUE THREAD

**Issue by piers7** [OpenLiveWriter/OpenLiveWriter#169]
So Blogger auth now works (for me at any rate). Woot! However - because there's no release notes on the OLW homepage - I found this out by accident.

What's the mechanism going forwards for finding out that new features or releases are available?

**Comment by kathweaver** [comment]
I would love to work on this -- and since I'm keeping up with testing hopefully a logical fit.

**Comment by rebornix** [comment]
As `OpenLiveWriter.Github.io` is using GitHub Page ( Jekyll ) for site generation and hosting, we can put the release notes in `_posts` folder and inject the posts for loop to anywhere we want to show the release notes, make sense?

You can take a look at [jekyll official site](https://github.com/jekyll/jekyll/tree/master/site/_posts) for reference.

**Comment by kathweaver** [comment]
@rebornix I was wondering if GitHub had a way of generating a list and if it could be linked to from the website.

We'd want it to add to the existing list as we go, rather than create a new list.

**Comment by rebornix** [comment]
Jekyll supports that, you can add a yaml front matter to the head of post like,

```
---
links:
  - link1
  - link2
  - link3
---
{% for link in page.links%}
  {{ link }}
{% endfor %}
```

**Comment by gep13** [comment]
@kathweaver GitHub has the concept of Releases, and these can be shown on a Project that implements them.  As an example, have a look at the releases page of the ChocolateyGUI project: https://github.com/chocolatey/ChocolateyGUI/releases.  Each time we release a version of that application, we create the release notes, and add them to a release on GitHub.  We do this automagically using a tool called [GitReleaseManager](https://github.com/GitTools/GitReleaseManager) which creates the notes based on which issues have been closed, and assigned to a milestone.  Since OpenLiveWriter is also using milestones to track the work that is being done, I personally think a combination of GitHub Releases and GitReleaseManager would be a good fit.

**NOTE:** Full disclosure, I am the current maintainer of the GitReleaseManager tool, so I might be a bit baised :smile:

**Comment by rebornix** [comment]
I have just sent out this pr https://github.com/OpenLiveWriter/OpenLiveWriter.Github.io/pull/15 for adding release notes to the website. @gep13 can we use GitReleaseManager to export markdown files from release notes then we can put it in the posts folder of the website.

The main reason of putting releases notes into `_posts` instead of binding a link to GitHub repo release tab is that maintainers may want to announce decisions/news other than release notes as well.

**Comment by gep13** [comment]
@rebornix yes, that was one of the main pieces of functionality that I wanted, as I want to embed the release notes into ChocolateyGUI as part of the release.  You can find that ability documented here: https://gitreleasemanager.readme.io/docs/export

**Comment by martinwoodward** [comment]
BTW - this is what we have in release notes ATM https://github.com/OpenLiveWriter/OpenLiveWriter/releases

**Comment by gep13** [comment]
@martinwoodward yip, and I think that this is a great start :smile: 

From my own, slightly obsessive, point of view, I like to know the individual features, bugs, improvements, etc, released as part of a new version.  Since you are already doing the hard work of tracking that information within the milestones (although I notice the 0.5.1 milestone is still open, should this be closed), it would be a small addition to your process to extract the issues (titles and number) into the release notes as well, so that people can see everything that was worked on.

**Comment by kathweaver** [comment]
@martinwoodward Can we just point to those release notes from the main page? Though weren't more improvements made than just that?

**Comment by rebornix** [comment]
@kathweaver it would be easy to put a link on the homepage to GitHub release notes. But I suppose users might expect a news page or Open Live Writer official blog on the main site. I've just proposed a pr https://github.com/OpenLiveWriter/OpenLiveWriter.Github.io/pull/15 to `openlivewriter.github.io` but the team's concerned about whether there are volunteers committed to help maintain the blog/news.

We may want to call out that OLW is welcome to Blog/News maintain besides code and documentation.

**Comment by kathweaver** [comment]
I would be happy to help maintain a news/blog post!

**Comment by rebornix** [comment]
@kathweaver put some thoughts in https://github.com/OpenLiveWriter/OpenLiveWriter.Github.io/pull/15 ?

**Comment by StephenCWLL** [comment]
I'll jump into this thread rather than start a new one that I was about to create about new releases.

There's no distribution of update news via the FB, twitter or newsletter channels. I know it's early days, and one could just bookmark https://github.com/OpenLiveWriter/OpenLiveWriter/releases and check it each day, but it'll be good if all releases or at least a link to the release story were posted through the 3 channels so everyone knows what's changed in an update and perhaps what on the horizon.
